credential-management/non-fully-active.https.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Credential Management API Test: non-fully active document</title>
+<link
+  rel="help"
+  href="https://github.com/w3c/webappsec-credential-management"
+/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<body></body>
+<script>
+  promise_test(async (t) => {
+    // Create the iframe, wait for it to load...
+    const iframe = document.createElement("iframe");
+
+    // ...wait for the iframe to load...
+    await new Promise((resolve) => {
+      iframe.addEventListener("load", resolve, { once: true });
+      iframe.src = "resources/iframe.html";
+      document.body.appendChild(iframe);
+    });
+
+    // Steal credentials container.
+    const { credentials } = iframe.contentWindow.navigator;
+
```

Comment:
As a sanity check, can we assert that a `get` works here?

Reply:
Depends what you mean by works? I could pre-check the abort signal first as confirmation.

Reply:
We should assert that `get` does not throw with `NotAllowedError` for some reason other than the one we want to test, before we detach the document.